tests/functional/translation/issues/00029.py

```diff
@@ -1,7 +1,7 @@
-#:: IgnoreFile(29)
 from nagini_contracts.contracts import *
 
 
 def test() -> None:
     r = [1, 2]
+    #:: ExpectedOutput(type.error:Encountered Any type. Type annotation missing?)
```

Comment:
Isn't this a MyPy concern? Should we worry about it?

Reply:
It seems that there was a Mypy problem at some point that caused Nagini to crash and Vytautas filed a bug because of that. But the Mypy problem is now fixed and everything does what it should. So I'd say the problem was never on our side but since there is an issue it doesn't hurt to have the regression test for this scenario.